specs/manualapprovalgate/manual-approval-gate.spec

```diff
@@ -0,0 +1,42 @@
+PIPELINES-28
+# ManualApprovalGate Pipelines operator specs
+
+## Approve Manual Approval gate pipeline: PIPELINES-28-TC01
+Tags: approvalgate, e2e, taskrun, sanity
+Component: Operator
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps:
+  * Apply
+    | S.NO | resource_dir                                          |
+    |------|-------------------------------------------------------|
+    | 1    | testdata/manualapprovalgate/manual-approval-gate.yaml |
+  * Validate manual approval gate deployment
```

Comment:
This is already done in installation test right?

Reply:
@ppitonak wdyt?

Reply:
In other tests (e.g. chains) we expect that the cluster is configured and usable. IMHO we can have the same assumption here.

Reply:
Okay. I will only keep validate deployment and remove the deployment yaml.